src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -31,6 +33,8 @@ public sealed partial class FileStatusList : GitModuleControl
         private readonly ToolStripItem _openInVisualStudioSeparator = new ToolStripSeparator();
         private readonly ToolStripItem _NO_TRANSLATE_openInVisualStudioMenuItem;
         private readonly CancellationTokenSequence _reloadSequence = new();
+        private readonly ToolStripItem _showAllDifferencesSeparator = new ToolStripSeparator() { Name = _showAllDifferencesItemName + "Separator" };
```

Comment:
```suggestion
        private readonly ToolStripItem _showAllDifferencesSeparator = new ToolStripSeparator() { Name = $"{_showAllDifferencesItemName}Separator" };
```

Reply:
Why?
It is compile-time constant.
And I would expect that a single concatenation of two (constant) strings is cheaper than string formatting.

Reply:
This seems like a premature micro optimisation. I trust the compiler to do the best here.
Interpolated strings are preferred to string concatenations.

Reply:
> This seems like a premature micro optimisation. I trust the compiler to do the best here.
> Interpolated strings are preferred to string concatenations.

1. I have just moved that code.
2. I was anticipating your performance argument with which you use to reason the switch to interpolated strings.
  Only in case of a simple concatenation, the `+` is slightly clearer to me than interpolation.

Though in order to have a simple coding rule, I have changed it as requested.

Reply:
Interpolation with constants does not have runtime overhead.

https://sharplab.io/#v2:C4LglgNgPgxg9gOwM7AAQAECMAGVTUC8qARABICmEEcxA3ALABQTWAdFgJwAUAJMQN5IAvgBpUAdzgAnCABMAhMQCUtIA===